railties/test/generators/app_generator_test.rb

```diff
@@ -290,6 +290,54 @@ def test_app_update_does_not_generate_bootsnap_contents_when_skip_bootsnap_is_gi
     end
   end
 
+  def test_app_update_preserves_skip_active_job
+    app_root = File.join(destination_root, "myapp")
+    run_generator [ app_root, "--skip-active-job", "--skip-active-storage", "--skip-action-mailer" ]
+
+    FileUtils.cd(app_root) do
+      config = "config/application.rb"
+      assert_no_changes -> { File.readlines(config).grep(/require /) } do
+        system("yes | TEST=hi bin/rails app:update")
```

Comment:
Sorry that was used to do some debugging when we were trying to figure out why Active Job was loaded even when `--skip-active-job` was used. It turns out that both [Active Storage](https://github.com/rails/rails/blob/9817c94fb41667adf8a247743d950fa7fbbfe698/activestorage/lib/active_storage/engine.rb#L5) and [Action Mailer](https://github.com/rails/rails/blob/9817c94fb41667adf8a247743d950fa7fbbfe698/actionmailer/lib/action_mailer/railtie.rb#L3) load the Active Job railtie.

Reply:
Oops, want to send a PR to fix that?

Reply:
https://github.com/rails/rails/pull/45354